flightControl/src/AvState.cpp

```diff
@@ -0,0 +1,262 @@
+#include "AvState.h"
+#include <string.h>
+#include <list>
+#include <sensors.h>
+#include "telecom.h"
+#include "PacketDefinition.h"
+
+
+AvState::AvState(const Thresholds& thresholds) : thresholds(thresholds)
+{
+    this->currentState = State::IDLE;
+}
+
+// destructor
+AvState::~AvState()
+{
+    currentState = State::IDLE;
+}
+
+
+// this function allows to get the current state of the FSM
+State AvState::getCurrentState()
+{
+    return currentState;
+}
+
+
+void sendError()
+{
+    //print in console the "error"
+    std::cout << "Error" << std::endl;
+}
+
+bool error()
```

Comment:
What is this for ?

Reply:
the error function is not implemented yet the above functionis obsolète